recipe/meta.yaml

```diff
@@ -105,7 +105,7 @@ requirements:
     - cudnn >=8.0.*,<9                      # [build_platform != target_platform and (not ((aarch64 or ppc64le) and (cuda_compiler_version or "").startswith("12")))]
     - nccl >=2.8,<3                         # [build_platform != target_platform]
     - cutensor >=1.4,<2                     # [build_platform != target_platform]
-    - cusparselt 0.2.0.*                    # [build_platform != target_platform and (linux64 or aarch64 or win) and (cuda_compiler_version or "").startswith("11")]
+    - cusparselt ~=0.2.0                    # [build_platform != target_platform and (linux64 or aarch64 or win) and (cuda_compiler_version or "").startswith("11")]
```

Comment:
The are effectively equivalent constraints. Please see [this doc for reference]( https://docs.conda.io/projects/conda-build/en/stable/resources/package-spec.html#package-match-specifications ). Quoting below for completeness

> `~=` | Compatibility Release | `~=0.5.3` is equivalent to `>=0.5.3,<0.6.0a`
> -- | -- | --